package.json

```diff
@@ -91,13 +91,13 @@
     "launch-editor-middleware": "^2.2.1",
     "mini-css-extract-plugin": "^0.4.0",
     "mkdirp": "^0.5.1",
-    "node-sass": "^4.5.0",
+    "node-sass": "^4.9.1",
     "optimize-css-assets-webpack-plugin": "^4.0.0",
     "postcss-loader": "^2.0.6",
     "prettier": "^1.10.2",
     "responselike": "^1.0.2",
     "rewire": "^2.5.2",
-    "sass-loader": "^6.0.6",
+    "sass-loader": "^7.0.3",
```

Comment:
If stylus isn't used anywhere, could you remove the stylus tooling from here and the webpack configurations?